### DISCUSSION TITLE
Trying to delete a github repository but I keep being redirected to import a repository

### DISCUSSION THREAD

**Discussion by notakamihe** [community/community#22189] [New to GitHub]
<p>So I have two empty repositories that I am trying to remove. When I open them I get redirected to a page that reads: “Import your project to Github” and “Your old repository’s clone URL”. The problem is that I don’t know my old repository’s clone URL so I cannot proceed. This issue is frustrating and unnecessary so I would appreciate any help I could get.</p>

**Comment by ghost** [comment]
<p>I am having the same issue. It is keeping me from using my chosen repository name.</p>

**Comment by ghost** [comment, marked as answer]
<p>I have found a solution. Open your repository, then change “import” in the URL to “settings”. This should allow you to delete the repository and start over.</p>

**Comment by notakamihe** [comment]
<p>Works like a charm. Thanks for the help!!!</p>
